src/Symfony/Component/HttpKernel/RebootableInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel;
+
+/**
+ * Rebotable extends the Kernel to allow it to reboot using a temporary cache directory.
```

Comment:
typo `Rebootable`. Anway, I would say `Allows the Kernel to be rebooted using ...`